### DISCUSSION TITLE
Is there a way to do custom style for react-calendar using styled-component from react-datepicker ?

### DISCUSSION THREAD

**Discussion by robert-truong** [wojtekmaj/react-date-picker#623] [General]
Hi all,

Is there a way to do custom style for react-calendar using styled-component from react-datepicker ?

**Comment by wojtekmaj** [comment]
Yes, kinda. Since styled-components only add className to root element being styled, you can write styles this way, but to style specific elements inside this root element you'll need to use nesting and rely on class based selectors.

Like this:

```js
const MyDatePicker = styled(DatePicker)`
  .react-calendar {
    // ...
  }
`;
